internal/controllers/utils/constants.go

```diff
@@ -7,20 +7,20 @@ const (
 
 // Base resource names
 const (
-	ORANO2IMSMetadata          = "metadata"
-	ORANO2IMSDeploymentManager = "deployment-manager"
-	ORANO2IMSResource          = "resource"
-	ORANO2IMSAlarmSubscription = "alarm-subscription"
-	ORANO2IMSAlarmNotification = "alarm-notification"
+	ORANO2IMSMetadata          = "metadata-server"
+	ORANO2IMSDeploymentManager = "deployment-manager-server"
+	ORANO2IMSResource          = "resource-server"
+	ORANO2IMSAlarmSubscription = "alarm-subscription-server"
+	ORANO2IMSAlarmNotification = "alarm-notification-server"
```

Comment:
Why are these being modified?

Reply:
linting was complaining about those in my env with below issue

```
internal/controllers/utils/constants.go:19:61: string `-server` has 5 occurrences, make it a constant (goconst)
        ORANO2IMSMetadataServerName          = ORANO2IMSMetadata + "-server"
```  

version in my env is
```
-> golangci-lint --version
golangci-lint has version 1.55.2 built with go1.21.3 from e3c2265f on 2023-11-03T12:59:25Z
```

Reply:
The solution to that, though, would be to add a serverSuffix const that gets used in place of "-server", rather than changing the base const value